src/components/components/Character.test.jsx

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+
+import Character from "./Characters";
+
+describe("testing", () => {
+  it("character", () => {
+    const mock = [
```

Comment:
this test only needs a character object instead of an array, the snapshot file is full of errors and doesn't match what would be expected.